src/wallet.js

```diff
@@ -0,0 +1,51 @@
+const ainUtil = require('@ainblockchain/ain-util')
+
+/**
+ * Derive an account(address,public key) from a private key
+ * @param {String} privateKey 
+ * @return {Strings} address, private_key, public_key 
+ */
+function fromPrivateKey(privateKey) {
+    let buf_privateKey = Buffer.from(privateKey)
+    let publicKey = ainUtil.privateToPublic(buf_privateKey)
+    return {
+        address : ainUtil.toChecksumAddress(ainUtil.bufferToHex(ainUtil.pubToAddress(publicKey))),
```

Comment:
Could you remove the space before colon (':') here and other places?

Reply:
Yeap. I did it.